### PR TITLE
Repush add shortdesk

### PR DESCRIPTION
**Pull request by mvr320**
There is some optimization still to be done, the new forloops will often find the same variables for the DB and can be optimized away.

This PR is first to see if the current code does still work.

The commits are done in such a way that the indentation can be followed, I'll rewrite the commits when this is approved to make more sense for the history.

### CODE REVIEW

File: webapp/src/Entity/Contest.php [hunk 1]
```diff
@@ -381,6 +381,11 @@ public function getShortname(): string
         return $this->shortname;
     }
 
+    public function getShortDesc(): string
```
Comment (nickygerritsen) [inline webapp/src/Entity/Contest.php:384]: I’d be more of a fan to use getShortDescription

File: webapp/tests/Unit/Controller/Jury/tmpTest.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
+webapp/tests/Unit/Controller/Jury/ContestControllerTest.php
```
Comment (nickygerritsen) [inline webapp/tests/Unit/Controller/Jury/tmpTest.php:1]: What’s this file?

File: webapp/src/Controller/BaseController.php [hunk 3]
```diff
@@ -191,70 +182,72 @@ protected function deleteEntity(
...
-                                    $dependentEntities = $this->getDependentEntities($table, $relations);
-                                    if (!empty($dependentEntities)) {
-                                        $dependentEntitiesReadable = [];
-                                        foreach ($dependentEntities as $dependentEntity) {
-                                            $parts                       = explode('\\', $dependentEntity);
-                                            $dependentEntityType         = $parts[count($parts) - 1];
-                                            $dependentEntitiesReadable[] = str_replace(
-                                                '_', ' ',
-                                                $inflector->tableize($inflector->pluralize($dependentEntityType))
+        foreach($entities as $entity) {
```
Comment (nickygerritsen) [inline webapp/src/Controller/BaseController.php:185]: Nit: missing space

File: webapp/src/Controller/BaseController.php [hunk 4]
```diff
@@ -137,28 +137,19 @@ protected function saveEntity(
...
+    protected function deleteEntities(
         Request $request,
         EntityManagerInterface $entityManager,
         DOMJudgeService $DOMJudgeService,
         EventLogService $eventLogService,
         KernelInterface $kernel,
-        $entity,
-        string $description,
+        array $entities,
         string $redirectUrl
```
Comment (nickygerritsen) [inline webapp/src/Controller/BaseController.php:153]: Is this to prepare for multidelete?

Comment (mvr320) [reply webapp/src/Controller/BaseController.php:153]: yes, as getting this right is hard so I rather do the backend logic first and thann have the frontend and the needed route for it.

File: webapp/tests/Unit/Controller/Jury/tmpTest.php (hunk 2 above)
Comment (mvr320) [reply webapp/tests/Unit/Controller/Jury/tmpTest.php:1]: This branch originally also had tests... in refactoring I moved these to this tmp file, should not be needed in this form

File: webapp/src/Entity/Contest.php (hunk 1 above)
Comment (mvr320) [reply webapp/src/Entity/Contest.php:384]: I agree, will fix it.

**Review by nickygerritsen — APPROVED** [review]

**Comment by mvr320** [comment]
@nickygerritsen I changed some things in the commits, but I think the tests if everything works for lists with not only 1 element can be done later when the multidelete is actually added.

**Pull request merged**